modules/ting_das/ting_das.module

```diff
@@ -0,0 +1,378 @@
+<?php
+
+/**
+ * @file
+ * Handle the integration of digital article service (DAS) into the CMS.
+ */
+
+/**
+ * Implements hook_das_menu().
+ */
+function ting_das_menu() {
+  $items = array();
+
+  $items['admin/config/ting/das'] = array(
+    'title' => 'Digital article service',
+    'description' => 'Configure access to DAS.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ting_das_admin_settings_form'),
+    'access arguments' => array('administer site configuration'),
```

Comment:
Perhaps a little bit lazy here compared to implement our own permissions?

Reply:
Why should we want to implement our own permission, when this is exactly what I want?

Reply:
If we do not care about permissions we might as well lock it down to roles instead if that is exactly what we want. Perhaps that is less important.